packages/components/src/components/list/list.stories.ts

```diff
@@ -152,6 +152,81 @@ export const simple = (args: ListStoryArgs): string => html`
   </calcite-list>
 `;
 
+export const focus_TestOnly = (): string => html`
```

Comment:
Can you remove the `_TestOnly` suffix? This is a legacy pattern that we'll remove with https://github.com/Esri/calcite-design-system/issues/9085.